docs/get-started/angular.md

```diff
@@ -0,0 +1,440 @@
+| `"port"`                     | Port used by Storybook. <br/> `"port": 6006`                                                                                                                                                     |
+| `"host"`                     | Set up a custom host for Storybook. <br/> `"host": "http://my-custom-host"`                                                                                                                      |
+| `"configDir"`                | Storybook configuration directory location. <br/> `"configDir": ".storybook"`                                                                                                                    |
+| `"https"`                    | Starts Storybook with HTTPS enabled. <br/> `"https": true` <br/> Requires custom certificate information.                                                                                        |
+| `"sslCa"`                    | Provides an SSL certificate authority. <br/> `"sslCa": "your-custom-certificate-authority"` <br/> Optional usage with `"https"`                                                                  |
+| `"sslCert"`                  | Provides an SSL certificate. <br/> `"sslCert": "your-custom-certificate"` <br/> Required for `https`                                                                                             |
+| `"sslKey"`                   | Provides an SSL key to serve Storybook. <br/> `"sslKey": "your-ssl-key"`                                                                                                                         |
+| `"smokeTest"`                | Exit Storybook after successful start. <br/> `"smokeTest": true`                                                                                                                                 |
+| `"ci"`                       | Starts Storybook in CI mode (skips interactive prompts and will not open browser window). <br/> `"ci": true`                                                                                     |
+| `"quiet"`                    | Filters Storybook verbose build output. <br/> `"quiet": true`                                                                                                                                    |
+| `"docs"`                     | Starts Storybook in [documentation mode](../writing-docs/build-documentation.md#preview-storybooks-documentation). <br/> `"docs": true`                                                          |
+| `"styles"`                   | Provide the location of the [application's styles](../configure/styling-and-css.md#importing-css-files) to be used with Storybook. <br/> `"styles": ["src/styles.css", "src/styles.scss"]` <br/> |
+| `"stylePreprocessorOptions"` | Provides further customization for style preprocessors resolved to the workspace root. <br/> `"stylePreprocessorOptions": { "includePaths": ["src/styles"] }`                                    |
```

Comment:
In theory, we have dedicated options for starting Storybook and building it. 

Here are the options listed, which can be used when Storybook gets built:
https://github.com/storybookjs/storybook/blob/next/code/frameworks/angular/src/builders/build-storybook/schema.json

Here are the options listed, which can be used when Storybook is started in `dev` mode:
https://github.com/storybookjs/storybook/blob/next/code/frameworks/angular/src/builders/start-storybook/schema.json

The list of options may change over time. You decide whether to hardcode it into the documentation or just link to the aforementioned resources.